Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -66,8 +66,10 @@ class TabLocationView: UIView {
     }
 
     lazy var urlTextField: URLTextField = .build { urlTextField in
+        /* Ecosia: removing obsolete implementation as we don't support 4S anymore
         // Prevent the field from compressing the toolbar buttons on the 4S in landscape.
```

Comment:
Very minor and no blocker: 
I confess I'm a bit confused with the comment within a comment. I understand the idea is that the outer comment "negates" the inner one, but in that case is there a reason to keep the inner one at all?

Reply:
Total refuse! Thanks for checking!